client/src/components/LoginSession/tests/LoginSessionContainer-test.js

```diff
@@ -118,10 +114,9 @@ describe('LoginSessionContainer', () => {
       submitting: true
     });
 
-    httpResolve({
-      error: true,
-      success: false,
-      message: 'horrible failure'
+    httpReject({
+      response: {
+        text: () => Promise.resolve(JSON.stringify({ success: false, message: 'horrible failure' })) }
```

Comment:
we no longer need `success: false` since that's been superseded by status codes?

Reply:
Yep